main.py

```diff
@@ -66,7 +66,11 @@ def change_read():
         book.save()
     else:  # If book ID not found in db
         print('That book is not found.')
-        change_read()  # restarts function for book ID input
+        option = input('Return to main menu? Y for main menu. ').upper()
+        if option == 'Y':
+            print()
+        else:
+            change_read()  # restarts function for book ID input
```

Comment:
If the function ends you'll be taken back to the main menu, this is probably more complex than needed. Also use the ui methods to get input and display messages